Gemfile

```diff
@@ -16,6 +16,7 @@ gem 'rubocop-performance', '~> 1.8', require: false
 gem 'rubocop-rake', '~> 0.5', require: false
 gem 'rubocop-rspec', '~> 2.1', require: false
 gem 'simplecov', '~> 0.19', require: false
+gem 'timecop', '~> 0.9'
```

Comment:
It would be good to consider organizing these gems using groups rather than comments indicating groups.

Reply:
For a rubygem the Gemfile is only for development purposes so groups would only benefit those scopes.